apis/apps/v1alpha1/apicast_types.go

```diff
@@ -155,7 +191,7 @@ type APIcastCondition struct {
 
 	// The Reason, Message, LastHeartbeatTime and LastTransitionTime fields are
 	// optional. Unless we really use them they should directly not be used even
```

Comment:
This reads a bit weird. Probably don't use we and consider explaining why they should not be used?

Reply:
This is not shown on the generated CRD files. It is purely a code comment.